File: lavugio-front/src/app/shared/models/user/driverRegistration.ts
export interface DriverRegistration {
    email: string;
    password: string;
    name: string;
    lastName: string;
    phoneNumber: string;
    address: string;

    vehicleMake: string;
    vehicleModel: string;
    licenseNumber: string;
    licensePlate: string;
    vehicleColor: string;
    vehicleType: string;

    passangerSeats: number;

    petFriendly: boolean;
    babyFriendly: boolean;
}

File: lavugio-mobile/app/src/main/java/com/example/lavugio_mobile/models/DriverRegistration.java
package com.example.lavugio_mobile.models;

public class DriverRegistration {
    private String email;
    private String password;
    private String name;
    private String lastName;
    private String phoneNumber;
    private String address;
    private String vehiclePlate;
    private String vehicleType;

    public String getEmail() { return email; }
    public void setEmail(String email) { this.email = email; }

    public String getPassword() { return password; }
    public void setPassword(String password) { this.password = password; }

    public String getName() { return name; }
    public void setName(String name) { this.name = name; }

    public String getLastName() { return lastName; }
    public void setLastName(String lastName) { this.lastName = lastName; }

    public String getPhoneNumber() { return phoneNumber; }
    public void setPhoneNumber(String phoneNumber) { this.phoneNumber = phoneNumber; }

    public String getAddress() { return address; }
    public void setAddress(String address) { this.address = address; }

    public String getVehiclePlate() { return vehiclePlate; }
    public void setVehiclePlate(String vehiclePlate) { this.vehiclePlate = vehiclePlate; }

    public String getVehicleType() { return vehicleType; }
    public void setVehicleType(String vehicleType) { this.vehicleType = vehicleType; }
}